cloudwatch/cloudwatch.go

```diff
@@ -545,6 +557,9 @@ func (output *OutputPlugin) setGroupStreamNames(e *Event) {
 }
 
 func (output *OutputPlugin) createStream(e *Event) (*logStream, error) {
+	if !output.autoCreateStream {
+		return nil, fmt.Errorf("error: Log Stream %s does not exist in the log group %s and can't be created as autoCreateStream is disabled", e.stream, e.group)
```

Comment:
nit: let's use the user recognizeable name for the config option, `auto_create_stream`